.github/workflows/image-push-master.yml

```diff
@@ -0,0 +1,89 @@
+name: "Push images on merge to master"
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build-and-push-image-operator:
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_NAME: ghcr.io/${{ github.repository }}
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Docker
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GH_IMAGE_REGISTRY_TOKEN }}
+
+      - name: Build and push sriov-network-operator
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: |
+            ${{ env.IMAGE_NAME }}:latest
+            ${{ env.IMAGE_NAME }}:${{ github.sha }}
+          file: ./Dockerfile
+
+  build-and-push-image-config-daemon:
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_NAME: ghcr.io/${{ github.repository }}-config-daemon
```

Comment:
It should be noted that this currently changes the image name to what was previously built for openshift.
however seeing that we are moving to a new container image registry and this project since it was moved to k8snetworkplumbingwg did not have a set of containers publically available, i think its OK to consider the new name.

i.e instead of : sriov-network-config-daemon we have sriov-network-operator-config-daemon.

the benefit of using `github.repository` as part of the name is that if forked it will push images to the forked repo container registry if enabled with no additional changes.

Alternative is to keep the original name and have it hardcoded here.